easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.2-GCC-8.3.0.eb

```diff
@@ -26,6 +26,7 @@ configopts = (
     "--without-alps "
     "--without-libfabric "
     "--without-lsf "
+    "--without-mxm "
```

Comment:
you are modifying an existing eb recipe, can you confirm that this is not currently deployed in (pre-)production?

Reply:
This eb recipe was present in master, but in the end was not deployed on the system.